### ISSUE TITLE
Segmentations Settings UI

### ISSUE THREAD

**Issue by JamesAPetts** [OHIF/Viewers#1391]
Ongoing work on the segmentation UI. For this issue we should address global segmentation settings.. This ticket is addressing part of [this comment](https://github.com/OHIF/Viewers/issues/1370#issuecomment-576729840):
>It would be nice to have buttons that control segment visibility (overlay, contour, both, opacity), similar to what we have in Slicer (see you already have plans for this in the figma sketch).

The per-segment settings (color/alpha/visibility) part of the request is not in the scope of this issue, and will be implemented later.

[This Figma template](https://www.figma.com/file/0CP9cmhHhDC8gQX5R9zvgo/OHIF-segmentation-side-panel?node-id=0%3A1) shows how the UI should look.

- Add a gear icon to the top right of the segmentation panel.
- Clicking a Gear icon changes the contents of the side panel to be a settings menu.
- Render segment outline: Checkbox, default Checked
  - When checked, a horizontal slider appears for Outline Opacity with values 0 - 1.0 increments of 0.05
  -  When checked a horizontal slider appears for Outline Width with values 1-5, increment 1 px.
- Render segment fill: Checkbox, default Checked
  - When checked, a horizontal slider appears for Fill Opacity with values 0 - 1.0, increments of 0.05

- Render inactive segmentations: Default off
  - When checked a horizontal slider appears for Opacity of inactive segmentations with values 0 - 1.0 increments of 0.05.

Clicking the back button should go back to the segmentation menu. When the segmentation menu is hidden and then reopened, the view should default to the segmentation menu, not the settings menu.

The UI should alter these settings which are consumed by the viewer components: https://github.com/cornerstonejs/cornerstoneTools/blob/master/src/store/modules/segmentationModule/defaultConfiguration.js

**Comment by salimkanoun** [comment]
Excellent, excellent, excellent !
Can't wait to work with you on PET/CT segmentation, when OHIF will render fused images.

**Comment by fedorov** [comment]
I see #1392 has been merged, and IDC sandbox is at the latest 3.8.0 release of OHIF Viewer, but it does not work for me. 

Here are the issues:

1) maybe this is expected, but if I open TCIA QIN-HEADNECK collection, I do not see a single study that shows SEG in the Modality column; if I search for SEG in the Modality column, I get 0 studies
2) if I open a study that includes SR modality (since I expect those should have SEG as well for this specific collection), I do not see any SEG in the left column (which perhaps is expected, although confusing), but if I toggle the "Segmentations" "thingy" in the upper-right corner, the list is empty.

Here's example study: https://idc-sandbox-000.firebaseapp.com/projects/chc-tcia/locations/us-central1/datasets/qin-headneck/dicomStores/qin-headneck/study/1.3.6.1.4.1.14519.5.2.1.2744.7002.780384232736549650569052637489

![image](https://user-images.githubusercontent.com/313942/76799150-6599d080-67a7-11ea-840a-cb08a947d4eb.png)

**Comment by fedorov** [comment]
![image](https://user-images.githubusercontent.com/313942/76799217-83ffcc00-67a7-11ea-9191-ff6b0f9a5f43.png)

**Comment by pieper** [comment]
I agree, it's still quite confusing.  It partly works (in some cases).  Only the segs for the selected series are shown.

If you select the CT (at the bottom of the series list) you can see the seg that is defined on it.

![image](https://user-images.githubusercontent.com/126077/76800615-6e3fd600-67aa-11ea-80bd-0d86fd708427.png)

And same with the PET.

![image](https://user-images.githubusercontent.com/126077/76800677-8dd6fe80-67aa-11ea-9f82-d7f3d3a025e3.png)

It's cool that they show up in the MPR!  (Changing the selected segmentation doesn't seem to update the view though.

![image](https://user-images.githubusercontent.com/126077/76800801-c971c880-67aa-11ea-8fa7-073b808effaa.png)

**Comment by fedorov** [comment]
Hopefully, once https://github.com/ImagingDataCommons/IDC-WebApp/issues/49 is resolved, we would be able to try to use the IDC portal for subsetting the content to be viewed, and it should be easier to test the latest features.

**Comment by fedorov** [comment]
@pieper how can I identify studies that have segmentations? Or this is something that can't be done with the Study List interface at all?

**Comment by pieper** [comment]
> @pieper how can I identify studies that have segmentations? Or this is something that can't be done with the Study List interface at all?

I don't think you can tell currently.

**Comment by fedorov** [comment]
I was playing with the interface, and I realized there are some hick-ups with displaying segmentations. It is basically not clear whether segmentations are available for a given study. Segmentations list shows as empty, and looks like one needs to load imaging series that has segmentations first before segmentations list is populated. And even after that there seems to be a delay in that list being populated. See screencast here:

https://www.screencast.com/t/g6wzeltT 

The URL for that screencast is here: https://idc-sandbox-000.firebaseapp.com/projects/idc-tcia/locations/us-central1/datasets/lidc-idri-seg-sr-dcm/dicomStores/lidc-idri-seg-sr-dcm/study/1.3.6.1.4.1.14519.5.2.1.6279.6001.131383203689189807643685075952&sa=D&ust=1586369794567000&usg=AFQjCNHVY4eaLmveYQCRHA98Hwr7lOXIVA

@pieper this is the LIDC issue we discussed elsewhere.
